pyCGM_Single/pycgm_embed.py

```diff
@@ -32,9 +32,10 @@
 # Temporary try/except to deal with relative import issue
 # This file should eventually be rewritten anyway
 try:
-    import pycgmStatic
-    import pycgmIO
-    import pycgmCalc
+
+    import pyCGM_Single.pycgmStatic as pycgmStatic
```

Comment:
can you put this in the exception instead of try?

I suspect it is trying to import IO and failing, then it does nothing in the exception.

Reply:
Okay, placed the imports in the exception